x-pack/plugins/reporting/common/constants/routes.ts

```diff
@@ -25,7 +25,7 @@ export const INTERNAL_ROUTES = {
   DOWNLOAD_CSV: prefixInternalPath + '/generate/immediate/csv_searchsource',
   GENERATE_PREFIX: prefixInternalPath + '/generate', // exportTypeId is added to the final path
 };
-
+export const internalAccess = 'elasticInternalOrigin=true';
```

Comment:
It looks like we don't need to export this in Reporting code. Where it's needed, we can do:
```
import { ELASTIC_INTERNAL_ORIGIN_QUERY_PARAM } from '@kbn/core-http-common';
```